server/service/handler.go

```diff
@@ -454,6 +454,10 @@ func attachFleetAPIRoutes(r *mux.Router, svc fleet.Service, config config.FleetC
 	he.WithAltPaths("/api/v1/osquery/log").
 		POST("/api/osquery/log", submitLogsEndpoint, submitLogsRequest{})
 
+	// orbit authenticated endpoints
+	oe := newOrbitAuthenticatedEndpointer(svc, logger, opts, r, apiVersions...)
+	oe.POST("/api/_version_/fleet/orbit/config", getOrbitConfigEndpoint, orbitGetConfigRequest{})
```

Comment:
Similar to osquery endpoints, orbit endpoints shouldn't be versioned.

Reply:
IIRC the only reason we needed the alt path was to not break old releases. So in this case we don't need them. Just one path `/api/fleet/orbit/*`.

Reply:
sounds good